ref/README.md

```diff
@@ -21,8 +21,32 @@ cmake -DCMAKE_BUILD_TYPE=<debug | release> ..
 make VERBOSE=1
 ```
 
+### Building the kernel (Systems without modules)
+
+```
+rm -rf build ; mkdir build ; cd build
+export CC=<gcc | icc>
+export FC=<gfortran | ifort> 
+cmake -DCMAKE_BUILD_TYPE=<debug | release> ..
+make VERBOSE=1
+```
+
+### Additional description for build configurations 
+
+Some alterations could potentially need to be made in regards to the CC and FC compilers  
+
+Confirm and identifiy which compiler will be used on the system.  
+
+* export CC= < appropriate compiler & version  > 
+* export FC= < appropriate compiler & version  >
+
+For Example : ```export CC=gcc-10```
+
+
 ## Testing the kernel
 
+Ensure system is quipped with LFS to run the .nl files in test/test_input 
```

Comment:
@christopherwharrop-noaa 
I wanted to throw this line in here so that the LFS step we encountered yesterday didn't fall through the cracks. I'll create the "Prerequisites" section with a list of Dependencies above the "Building the kernel" section.

Reply:
@robgonzalezpita - Agree it is needed.  Since it is a software dependency, I think moving it to the top, along with other dependencies, will help make it clear what the user needs before going any further.

Reply:
# Building the kernel
 
## Prerequisites

### Dependencies 
* NetCDF 
* cmake 
* git-lfs

## Systems that use 'modules' to manage software

This kernel uses an out-of-source cmake build, meaning that the build must be done in
directory that is not in the source tree.

The build procedure is (from the directory containing this file):

```
rm -rf build ; mkdir build ; cd build
module load <compiler> netcdf cmake
export CC=<gcc | icc>
export FC=<gfortran | ifort>
cmake -DCMAKE_BUILD_TYPE=<debug | release> ..
make VERBOSE=1
```

## Systems without 'modules' to manage software

```
rm -rf build ; mkdir build ; cd build
export CC=<gcc | icc>
export FC=<gfortran | ifort>
cmake -DCMAKE_BUILD_TYPE=<debug | release> ..
make VERBOSE=1
```

Reply:
Just noticed that these comments support .md, so here is an edited draft of how I restructured the beginning.